src/main/kotlin/com/deviceinsight/helm/PackageMojo.kt

```diff
@@ -95,11 +96,13 @@ class PackageMojo : ResolveHelmMojo(), ServerAuthentication {
 
 			processHelmConfigFiles(targetHelmDir)
 
-			if (majorHelmVersion() < 3) {
-				executeCmd("$helm init --client-only --stable-repo-url $stableRepoUrl")
+			val helmAddFlags = if (isHelm2) emptyArray() else arrayOf("--force-update")
```

Comment:
Not sure about this one – it's potentially destructive, so we either have to make it a major release (breaking change) with very clear documentation about this, or we make it a parameter, so people have to opt-in to the new behavior.

Reply:
Yeah, you are right. I've introduced the `forceAddRepos` flag which defaults to `false`